Assets/Scripts/Level generation scripts/NavigationGraph.cs

```diff
@@ -99,7 +104,7 @@ public bool CheckNewLinks(List<Node> cluster, bool canCreateNew = true)
                 // continue;
 
                 // if dist > nodesMaxDistance, add in between nodes
-                if (dist > NavigationGraph.maxLinkLength && dist < NavigationGraph.nodesMaxDistance && canCreateNew)
+                if (dist > NavigationGraph.maxLinkLength && dist < NavigationGraph.nodesMaxDistance && canCreateNew && false) // Disable this for now
```

Comment:
why disable?

Reply:
This would create more intermediate nodes in between long links. If there is 2 nodes a bit far from each other, it add nodes in between to split the big link in smaller ones
I just disabled it to generate the graph quicker, and because I think somebody said that bigger links are better

Should i enable it back ?